core/repository/sail/src/main/java/org/eclipse/rdf4j/repository/sail/SailRepository.java

```diff
@@ -166,6 +166,7 @@ protected void shutDownInternal()
 		throws RepositoryException
 	{
 		try {
+			logger.debug("Sail: {}", sail);
```

Comment:
Rephrase to "Shutting down Sail: {}" - btw does sail.toString() give a useful, readable message?